internal/fs/inode/base_dir.go

```diff
@@ -263,3 +263,9 @@ func (d *baseDirInode) IsUnlinked() bool {
 
 func (d *baseDirInode) Unlink() {
 }
+
+// HasNoSupportedObjectsInSubtree checks and returns true if this directory
```

Comment:
This comment can just be in the interface. Here we can just say that baseDirInode can't have unsupported objects

Reply:
done